alacritty/src/message_bar.rs

```diff
@@ -341,6 +350,23 @@ mod tests {
         ]);
     }
 
+    #[test]
+    fn wrap_with_unicode() {
+        let input = "ab\nc 👩d e fgh";
+        let mut message_buffer = MessageBuffer::new();
+        message_buffer.push(Message::new(input.into(), MessageType::Error));
+        let size = SizeInfo::new(7., 10., 1., 1., 0., 0., false);
+
+        let lines = message_buffer.message().unwrap().text(&size);
+
+        assert_eq!(lines, vec![
+            String::from("ab  [X]"),
+            // the "e" would fit in this line but 👩 width is 2
```

Comment:
I don't think it's necessary to document this. It should be very clear because your editor shows you exactly how big it is. It's also a bit incorrect, because it wouldn't fit and it doesn't because this isn't just a single width character.